src/trx/decoder/decoder.ts

```diff
@@ -59,7 +69,7 @@ export class TrxDecoder implements Decoder<FunctionCallExplanation> {
     const abiEncodedArgs = bufferToHex(data);
     const decodedArguments = TronWeb.utils.abi.decodeParams(names, types, abiEncodedArgs, true);
 
-    const args = [];
+    const args: any[] = [];
     for (let i = 0; i < inputs.length; i++) {
       const { name, type } = inputs[i];
       const decodedArgument = decodedArguments[name];
```

Comment:
Also looks like duplicated code

Reply:
formatValue() is an implementation that depends on the chain, we could pass that function as a parameter but since we dont know how will be the future implementations of 'decode()' its pretty weird.